open_academy/models/__init__.py

```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
+
+from . import models
```

Comment:
```suggestion
from . import models
```
add newline at the end of file